azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/boot/ApplicationInsightsModuleConfiguration.java

```diff
@@ -84,6 +85,15 @@ public DeviceInfoContextInitializer deviceInfoContextInitializer() {
         return new DeviceInfoContextInitializer();
     }
 
+    /**
+     * Bean for CloudInfoContextInitializer
+     * @return instance of {@link CloudInfoContextInitializer}
+     */
+    @Bean
+    public CloudInfoContextInitializer cloudInfoContextInitializer() {
```

Comment:
By the way @littleaj reduce the visibility of this to default instead of public. See #718 PR where I have reduced the visibility of all beans as they are not needed outside.

Reply:
will do.